src/apis/product/index.ts

```diff
@@ -0,0 +1,32 @@
+import type { ProductWithInfo } from '@/types/product';
+
+import instance from '../instance';
+import API from '../path';
+
+export interface GetThemesParams {
+  themeKey: string;
+  pageToken?: string;
+  maxResults?: number;
+}
+export const getProductWithTheme = async ({
+  themeKey,
+  pageToken,
+  maxResults,
+}: GetThemesParams): Promise<ProductWithInfo> => {
+  console.log(maxResults);
+  console.log(themeKey);
+  const params = {
+    pageToken,
+    maxResults,
+  };
+
+  if (pageToken) params.pageToken = pageToken;
```

Comment:
요렇게 쓰면 코드를 간소화 시킬 수 있겠네요

```suggestion
  const params = {
  pageToken,
  maxResults,
  ...(pageToken && { pageToken })
};
```

Reply:
오.. 수정하도록 하겠습니다!